pkg/deploy/dev-workspace/dev_workspace.go

```diff
@@ -172,6 +179,31 @@ func ReconcileDevWorkspace(deployContext *deploy.DeployContext) (bool, error) {
 	return true, nil
 }
 
+func checkIfCheIsInstalledInSingleNamespacesMode(deployContext *deploy.DeployContext) bool {
+	// If clusterserviceversions resource doesn't exist in cluster DWO as well will not be present
+	if !util.HasK8SResourceObject(deployContext.ClusterAPI.DiscoveryClient, ClusterServiceVersionResourceName) {
```

Comment:
Is it possible not to have CSV CRD on OpenShft ?

Reply:
Right, I will remove it.

Reply:
OCP 3.11 don't have CSVs

Reply:
We don't support OCP 3.11, so we can safely remove these lines
https://github.com/eclipse-che/che-operator/pull/1065/files#diff-818e3c93d561e8cbbbafa8a7ddee39fa909da19f9034de733b1ce5d747267791L118-L120